lib/publishing.ts

```diff
@@ -281,8 +291,11 @@ export class AssetPublishing implements IPublishProgress {
     if (existing) {
       return existing;
     }
+    if (this.options.quiet && this.options.subprocessOutputDestination == 'stdio') {
+      this.options.subprocessOutputDestination = 'ignore';
+    }
```

Comment:
This is still leaving the option open for some nonsensical combinations, no?

`{ quiet: true, subprocessOutputDestination: 'publish' }` ?
`{ quiet: false, subprocessOutputDestination: 'stdio' }` ?

What we usually do when replacing a property with another one is to say that you're not allowed to pass both.

```ts
if (this.options.quiet !== undefined && this.options.subprocessOutputDestination) {
  throw new Error('No way José');
}

const subprocessOutputDestination =  this.options.subprocessOutputDestination ?? (this.options.quiet ? 'ignore' : 'stdio');
```